solutions/cses-1139.mdx

```diff
@@ -0,0 +1,113 @@
+---
+id: cses-1139
+source: CSES
+title: Distinct Colors
+author: Timothy Gao
+---
+
+## Editorial
+
+Let us consider the Euler Tour of the tree. We flatten the tree into an array, where each node corresponds to a range in this array. Now, we are essentially tasked the find the number of distinct values in a range $N$ times(once for each node).
+
+There are a variety of ways to approach the problem from here.
```

Comment:
Take out this line - it's redundant

Reply:
Also, you say this but only list 1 solution

Reply:
I'm going to add 2 more approaches in later pull requests.

Reply:
(Small to Large Merging and Mo's Algorithm)

Reply:
> (Small to Large Merging and Mo's Algorithm)

for small to large merging, link the module (the solution is already there)